composer.json

```
@@ -29,11 +29,11 @@
         "phpcr/phpcr-implementation": "^2.1",
         "phpcr/phpcr-utils": "^1.3.0",
         "doctrine/instantiator": "^1.0.1",
-        "symfony/console": "^3.3 || ^4.0"
+        "symfony/console": "^3.3 || ^4.0 || ^5.0"
```

Because master is going to be version 2.0, I would like to see the versions bumped to the current supported versions.
```suggestion
        "symfony/console": "^3.4 || ^4.3 || ^5.0"
```